requirements.txt

```diff
@@ -1,6 +1,6 @@
 colorama==0.3.6
 docopt==0.6.2
-Jinja2
+Jinja2==2.8
```

Comment:
Addressed in https://github.com/metacloud/molecule/pull/150/commits/f23d2025ffa11df7470e3babf476cdc69308a01a.